.github/workflows/on-doc-v2-merge.yml

```diff
@@ -1,25 +0,0 @@
-name: Docs v2 Publish (merge)
```

Comment:
No longer needed?

Reply:
On second thought I think we don't.

I created it but it doesn't work, and since we are near merging the v2 branch I don't want to invest time in figuring out what's happening with the jq & `versions.json` file.

We are going to publish the migration guide (which I'll finish soon) on `main` and with an item in the navbar so customers have a chance to see it soon.

For changes that are specific to utility pages, we can just use v2 and use local preview for the next week or two before we finally merge v2.